image/bin/my_init

```diff
@@ -28,6 +28,30 @@ terminated_child_processes = {}
 
 _find_unsafe = re.compile(r'[^\w@%+=:,./-]').search
 
+try:
+    basestring
+except NameError:
+    basestring = (str, bytes)
+
+def to_bool(s):
```

Comment:
Pyramid's asbool function is a cleaner implementation of this, I believe:
https://github.com/Pylons/pyramid/blob/master/pyramid/settings.py#L6